src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/stores/ListStore.js

```diff
@@ -511,6 +517,11 @@ export default class ListStore {
                 this.initialSelectionIds = undefined;
             }
         })).catch((response) => {
+            if (response.name === 'AbortError') {
```

Comment:
i am a bit hesitant to rely on the `name` here.. isnt there at least a constant somewhere that we could use?

Reply:
also why dont you set `this.pendingRequest` to undefined in this case? :

Reply:
At least the `name` attribute is safe to use, because it is standardized, as documented in [MDN](https://developer.mozilla.org/en-US/docs/Web/API/AbortController/AbortController) and [Google Developers](https://developers.google.com/web/updates/2017/09/abortable-fetch#reacting_to_an_aborted_fetch).

Reply:
Regarding setting `this.pendingRequest`. We could do that, but I think it is safer not to. The only place we abort the request is right before we are assigning a new value to `this.pendingRequest` anyway. So assigning another value does not make too much sense, since it will be reassigned immediately. Additionally I am not sure if that could cause some kind of race condition, therefore I think it is safer not to do it.